Source/Timeline/EventView.swift

```diff
@@ -29,6 +29,8 @@ open class EventView: UIView {
     }
   }
 
+  public var extraData: Any?
```

Comment:
userInfo makes sense!

Do you think it should also be a [String : Any] dictionary (as UserInfo usually is in Apple's frameworks) or leave it as an Any?